de-lib/src/main/java/org/iplantc/de/apps/widgets/client/view/editors/arguments/converters/SplittableToFileConverter.java

```diff
@@ -31,10 +32,11 @@ public File convertModelValue(Splittable object) {
         if (object == null) {
             return null;
           }
-        if(object.isString()){
+        if (object != null && object.get("path") != null && !Strings.isNullOrEmpty(object.get("path")
```

Comment:
We don't have to check `object != null` here, because there is a `null` check right above this. It might also be slightly more readable to refactor `object.get("path").asString()` into a variable, which could also be reused in the `file.setPath` below:
```java
        String path = object.get("path") != null ? object.get("path").asString() : "";
        if (!Strings.isNullOrEmpty(path)) {
            File file = factory.file().as();
            file.setPath(path);
            return file;
        }
        return null;
```

Reply:
👍 . Actually i just realized i missed this fix for `SplittableToFolderConverter`. Will add that too.